input/elasticapm/internal/modeldecoder/v2/model.go

```diff
@@ -308,6 +308,8 @@ type contextServiceOrigin struct {
 }
 
 type contextServiceAgent struct {
+	// ActivationMethod of the APM agent capturing information.
+	ActivationMethod nullable.String `json:"activation_method" validate:"maxLength=1024"`
```

Comment:
Why is the field also added to the `context.service.agent`? I don't think this is necessary, as my understanding is that agents would send it in the `metadata` of the request.

Reply:
Right, I missed the keyword "new metadata field" in the issue. At the time of reading it, I didn't know it implies implementation details. Will double check vs issue description before asking for review next time.

Reply:
no worries